src/content/docs/alerts-applied-intelligence/transition-guide.mdx

```diff
@@ -1,15 +1,79 @@
 ---
-title: Alerts and applied intelligence transition guide 2022
+title: Alerts and applied intelligence transition guide
 tags:
   - Alerts and Applied Intelligence
   - New Relic Alerts
   - EOL
   - Transition guide
 metaDescription: 'Resources to make a successful transition as new functionalities in New Relic One replace alerting functionalities reaching end of life in 2022.'
+redirects:
+  - /docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-ai-transition-guide-2022 
 ---
 
 Over the course of several months during 2022, new functionalities in New Relic One will replace alerting functionalities reaching end of life (EOL). We are providing these early notices so you can plan accordingly and make a smooth transition.
 
+## What's changed [#changed]
+
+Starting in late 2021, we've reorganized the navigation UI for alerts and applied intelligence. Although the alerts and applied intelligence navigation has changed pretty dramatically, all the features you use are still there.
+
+We're slowly rolling out this update to all of our users.
+
+![A screenshot of the new alerts and applied intelligence left navigation.](./images/alerts-ai-categories.png "The new alerts and applied intelligence left navigation")
+
+<figcaption>The new UI includes these new categories.</figcaption>
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="nav-changes"
+    title="An overview of alerts and applied intelligence nav changes"
+  >
+
+  ## Analyze [#analyze]
+
```

Comment:
Maybe an intro sentence about the kinds of things you might want to analyze, and, ta da! The UI helps you do exactly those things! Like, why do I care about open, closed, muted violations? What is the timeframe for most recent issues -- can I change that?

Reply:
Yup, I've gone through and added some intro sentences to each of these sections.